core/Symbols.cc

```diff
@@ -1208,9 +1208,12 @@ void Symbol::addLoc(const core::GlobalState &gs, core::Loc loc) {
         return;
     }
 
-    // We shouldn't add locs for <root>, otherwise it'll end up with a massive loc list (O(number of files)).
-    // Those locs aren't useful, either.
+    // We shouldn't add locs for <root> or <PackageRegistry>, otherwise it'll end up with a massive loc list (O(number
+    // of files)). Those locs aren't useful, either.
```

Comment:
ooc, did you make this change pre-emptively, or was there a test failure that forced you to make this change?

Reply:
Pre-emptively.